src/cfg/structured.rs

```diff
@@ -33,7 +33,21 @@ impl Display for StructuredProgram {
     }
 }
 
-#[derive(Debug)]
+impl StructuredProgram {
+    pub fn to_program(&self) -> Program {
```

Comment:
Can't comment on program.rs because it's not in the diff, but should `Program` be renamed to `UnstructuredProgram` for clarity?

Reply:
Program is a bril type, I could make an alias but that would be confusing. Otherwise this would be a good idea